pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -16,18 +17,23 @@
 , libX11
 , mangohud32
 , vulkan-headers
+, glew
```

Comment:
tiny formatting nit: The existing attributes aren't sorted alphabetically, they're sorted by usage. For consistency it might be better just to put all the new `buildInputs` after `spdlog`.